config/config.go

```diff
@@ -60,6 +61,7 @@ func Get() (*Configuration, error) {
 		EnableDetachDataset:        false,
 		EnablePermissionsAuth:      false,
 		EnableObservationEndpoint:  true,
+		KafkaVersion:               "1.0.2",
```

Comment:
Does this need to be added to the secrets as well?

Reply:
This is the Kafka version that is used in the environments at the moment (1.0.2). If we update Kafka at some point, we can add `KAFKA_VERSION=x.y.z` to the secrets of services that use Kafka at that point. Do we need to add them if the default version matches the one that we need at this point? (I agree it would be an easy way to know what we need to modify in the future). If we do, then we should update the secrets for other services that have been updated to dp-kafka v2 too.

Reply:
Sounds sensible - I was just asking the question to make sure it had been considered. I'm happy with your explanation so will approve now.